spec/dummy-rails-app/config/routes.rb

```diff
@@ -1,5 +1,15 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  require 'sidekiq/web'
+
+  # In order to force sidekiq to use the rails app's session,
+  # we need to disable the Sidekiq's session.
+  Sidekiq::Web.disable(:sessions)
```

Comment:
I feel like this should probably be in a `sidekiq.rb` initializer (even though it will be the only thing in there). It's not really to do with routing.

Reply:
Agreed